src/Controllers/SuperadminController.php

```
@@ -169,6 +172,93 @@ public function subscriptions(): void
+        }
+
+        $this->response->redirect('/superadmin/packages');
+    }
+
+    public function rejectPackageChange($id): void
+    {
+        $user = $this->auth->user();
+        PackageChangeRequest::reject($this->db, (int) $id, (int) $user['id']);
+        $_SESSION['flash_message'] = 'Package change rejected.';
+        $this->response->redirect('/superadmin/packages');
+    }
```

PHPMD (2.15.0)

[error] rejectPackageChange accesses the super-global variable $_SESSION. (undefined)

(Superglobals)